README.md

```diff
@@ -93,196 +53,134 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
+
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Register and log in, so that the data is private to them]**
+- **[Introduce new transactions associated with a category]**
+- **[See the money spent on each category]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://budget-app-56iz.onrender.com)
+
+- [Video Presentation](https://www.loom.com/share/b7e5378aa77a4fd584c551e77cedf6a3)
```

Comment:
- [ ] Great job deploying your app to render however, I get this page when I try to access the registration or login page 👇🏻 . please, let's make sure our app is properly deployed and all functions are working as expected.

![image](https://github.com/JacintaNgwu/Budget-App/assets/79058364/55a517c6-5d97-4938-9356-e5daa26e06f6)